src/main/kotlin/com/team4099/robot2022/Robot.kt

```diff
@@ -15,6 +16,9 @@ import kotlin.math.pow
 
 object Robot : TimedRobot() {
   val robotName: Constants.Tuning.RobotName
+  val autonomousSelector: AutonomousSelector? = null
```

Comment:
can you not just do `val autonomousSelector = AutonomousSelector()` here?

Reply:
if not I think you want to look into `lateinit` so you don't have to make this an optional, should avoid nullable things wherever possible